src/main/groovy/org/mockito/release/internal/gradle/AllContributorsFetcherTask.java

```diff
@@ -0,0 +1,64 @@
+package org.mockito.release.internal.gradle;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.OutputFile;
+import org.gradle.api.tasks.TaskAction;
+import org.mockito.release.notes.contributors.*;
+
+import java.io.File;
+
+/**
+ * Fetch data about all project contributors and store it in file. It is used later in generation pom.xml.
+ */
+public class AllContributorsFetcherTask extends DefaultTask {
+
+    private static final Logger LOG = Logging.getLogger(AllContributorsFetcherTask.class);
+
+    @Input private String repository;
+    @Input private String readOnlyAuthToken;
+    @Input private boolean skipTaskExecution;
```

Comment:
I suspect we don't really need this field. You can use existing Gradle Task API to do this:

```java
//during Gradle's configuration phase:
someTask.setEnabled(false);

//or, alternatively hook up the task dependencies conditionally during configuration phase:
if (...) {
  someTask.dependsOn(otherTask);
}

//during Gradle's task execution phase
someTask.onlyIf(new Spec() { ... })
```